common/changes/@microsoft/rush/danade-preventManualLockfileChanges_2020-02-08-05-33.json

```diff
@@ -0,0 +1,11 @@
+{
+  "changes": [
+    {
+      "packageName": "@microsoft/rush",
+      "comment": "Add option to validate the lockfile was not manually edited, failing \"rush install\" if changes are detected.",
```

Comment:
This is a little difficult to understand without context.  Maybe we could just say `Add a new rush.json setting "preventManualShrinkwrapChanges" which prevents against accidental mistakes in pnpm-lock.yaml`